manifest.json

```diff
@@ -45,6 +58,16 @@
 				"3511083906": "The wife of Gregory and daughter-in-law to Madam Carlisle, Emma is a suspect in the murder of Zachary Carlisle.<br><br>She claims to have gone to bed with an \"awful migraine\" when Edward and Gregory had a drink around 8 PM last night. Emma says Zachary's company was always \"awkward\" due to his all consuming fascination with plants. She says she considers Madam Carlisle at least partially responsible for his supposed suicide because she allowed Zachary to become a recluse.<br><br>Lastly, Emma states that she has seen nothing out of the ordinary except that Zachary appeared depressed after the apparent loss of his sister Madam Carlisle."
 			}
 		},
+		"008322D98E1CFE04": {
+			"english": {
+				"611928695": "Old garden shears."
+			}
+		},
+		"00873D29DB58C88A": {
+			"english": {
+				"594769380": "A vintage Branson MD-2 condenser microphone. This rare model was removed from the market in the 1960s after causing a number of electric shock fatalities. Apparently, a production defect makes the MD-2 short-circuit at high voltages."
+			}
+		},
```

Comment:
This feels to me like an opinionated change, -1 on adding.

Reply:
I do have to say I agree. I think I lost track of it a bit in all the shuffling around, but the comma really does seem like a matter of style. It's not an Oxford comma, but the same principle applies.

Reply:
If I remember from Discord, you said it was written like as if someone was speaking, which doesn;t always translate well to text, hence the request to change it from Stark on Discord. I honestly don't mind either way.